src/gpu/intel/gemm/jit_xe_hp_systolic.cpp

```diff
@@ -211,7 +211,7 @@ bool xe_hp_systolic_t::pd_t::use_nocopy() {
 
     bool xehpc = (arch >= compute::gpu_arch_t::xe_hpc);
 
-    if (any_prepacked_ || (packed_a_ && packed_b_)) return false;
+    if (any_prepacked_) return false;
```

Comment:
Can you explain what this change is doing? My assumption was that `xe_hp_systolic_t` doesn't support double-packed inputs - is this just an obsolete check?